### PR TITLE
update home page

### PR DESCRIPTION
**Pull request by chrismgeorge**
update home page based on new design

### CODE REVIEW

**Comment by chrismgeorge** [comment]
viewable at modelviewer.dev/home.html

File: packages/modelviewer.dev/home.html [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+              </example-snippet>
+            </div>
+          </div>
+
+        <div id="section-documentation">
+          <h3 class="grouping-title grouping-title-new border-bottom">Documentation</h3>
+          <ol class="new-list-example">
+            <li>
+              <h4>
+                <span class="attribute">&lt;model-viewer&gt;</span>’s <a href="docs/index.html">documentation</a> and <a href="examples/loading/index.html">examples</a>
```
Comment (elalish) [inline packages/modelviewer.dev/home.html:75]: I think they know it's model-viewer by now. How about "API documentation & examples"?

File: packages/modelviewer.dev/home.html [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+  <a id="button-github" class="icon-github icon-button" href="https://github.com/google/model-viewer"></a>
+
+  <div class="sample">
+    <div id="demo-container" class="demo"></div>
+    <div class="content">
+      <div class="wrapper">
+        <div class="heading">
+          <div class="lockup zero-interaction"><div class="icon-button icon-modelviewer-black"></div><h1><span class="attribute">&lt;model-viewer&gt;</span></h1></div>
+          <h3 class="slogan" style="font-weight:400;">Easily display interactive 3D models on the web & in AR</h3>
```
Comment (elalish) [inline packages/modelviewer.dev/home.html:51]: This seems to be getting line-wrapped early.

File: packages/modelviewer.dev/styles/examples.css [hunk 3]
```diff
@@ -311,6 +311,11 @@ h1 span {
   margin: var(--snippet-offset);
 }
 
+.content .quick-start-example example-snippet {
```
Comment (elalish) [inline packages/modelviewer.dev/styles/examples.css:314]: On android, the h4 text seems to be about the same size as the code snippet text (good). On my mac, it's quite a bit larger, nearly as big as the h3 size. Is this a `px` vs `em` thing?

File: packages/modelviewer.dev/home.html (hunk 1 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/home.html:75]: k

File: packages/modelviewer.dev/home.html (hunk 2 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/home.html:51]: that's how it currently is, do you want that to just stretch out all the way?

File: packages/modelviewer.dev/styles/examples.css (hunk 3 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/styles/examples.css:314]: **"it's"** quite a bit larger -- what exactly is the it's referring to? the code snippet or h4?

if you are referring to h4, this is due to the sizes being scaled as the screen sizes change (you'll see this if you scale the screen width down)

this is currently what happens on dev as well and is due to: @media only screen and (max-width: 800px), inside it, the text is changing for any window smaller than 800 px (like an android)

File: packages/modelviewer.dev/home.html (hunk 2 above)
Comment (elalish) [reply packages/modelviewer.dev/home.html:51]: Yeah, does that seem reasonable to you?

File: packages/modelviewer.dev/styles/examples.css (hunk 3 above)
Comment (elalish) [reply packages/modelviewer.dev/styles/examples.css:314]: Yeah, the h4 is bigger on desktop, relative to both the h3 and the code snippet. Seems like they should all be scaled together, maintaining relative size, more or less for mobile vs desktop.

Comment (chrismgeorge) [reply packages/modelviewer.dev/styles/examples.css:314]: I don't think that's exactly right. So em is apparently: Relative to the font-size of the element (2em means 2 times the size of the current font).

So, the difference between smaller numbers is more obvious than higher numbers, i.e. 98 and 100 are close but 1 and 3 would be drastic, relatively. 

The desktop has----------------1.4em for h3 and 1.2em for h4
The sub-800px width-----------1.2em for h3 and 1.0em for h4

So, yes the desktops would look closer in size, but the h4 isn't bigger.

File: packages/modelviewer.dev/home.html (hunk 2 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/home.html:51]: Yeah, it doesn't look too different, and keeps everything consistent

File: packages/modelviewer.dev/styles/examples.css (hunk 3 above)
Comment (elalish) [reply packages/modelviewer.dev/styles/examples.css:314]: Okay, no biggy.

Comment (chrismgeorge) [reply packages/modelviewer.dev/styles/examples.css:313]: kk

File: packages/modelviewer.dev/home.html (hunk 2 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/home.html:51]: done

File: packages/modelviewer.dev/home.html (hunk 1 above)
Comment (chrismgeorge) [reply packages/modelviewer.dev/home.html]: done

**Review by elalish — APPROVED** [review]
Thanks! Want to ping Ricardo for a review once this deploys? Would be nice to talk to him about it tomorrow.

**Comment by chrismgeorge** [comment]
> Thanks! Want to ping Ricardo for a review once this deploys? Would be nice to talk to him about it tomorrow.

yeah, if he's back from vacation

**Pull request merged**